DevEdu.API/DevEdu.API/Controllers/GroupController.cs

```diff
@@ -62,30 +59,17 @@ public void ChangeGroupStatus(int groupId, int statusId)
         //add group_lesson relation
         // api/Group/{groupId}/lesson/{lessonId}
         [HttpPost("{groupId}/lesson/{lessonId}")]
-        public string AddGroupLessonReference(int groupId, int lessonId)
+        public void AddGroupLessonReference(int groupId, int lessonId)
         {
-            return (lessonId + groupId).ToString();
+            _groupRepository.AddGroupLesson(groupId,  lessonId);
         }
 
         // api/Group/{groupId}/lesson/{lessonId}
         [HttpDelete("{groupId}/lesson/{lessonId}")]
-        public string RemoveGroupLessonReference(int groupId, int lessonId)
+        public void RemoveGroupLessonReference(int groupId, int lessonId)
         {
-            return (lessonId - groupId).ToString();
+            _groupRepository.RemoveGroupLesson(groupId, lessonId);
         }
 
-        // api/Group/{groupId}/material/{materialId}
```

Comment:
why have you deleted all the rest methods?) it might cause some conflict...
could you revert this?